### PR TITLE
Package MarkerMap for local installation

### PR DESCRIPTION
**Pull request by WilsonGregory**
## Changes
- add MIT license, fix for: https://github.com/Computational-Morphogenomics-Group/MarkerMap/issues/18
- add files to allow creating a package for the code
- update the benchmark_k script to import the package without having to add it to the path
- update the notebooks to import the package without having to add it to the path
- I did not update the notebooks Array_of_Markers and DemoSimpleInterface because they are on colab, so they need to be able to install the package from pip before we can swap over (as far as I am aware)

## Testing
- run through select portions of the notebooks to make sure the imports work correctly
- run through the benchmark script to make sure the imports work there
- future testing: Do a fresh install following the instructions to make sure they work correctly
- ran unit tests

## Doc Changes
- some updates to the readme, specify locally installing the package

### CODE REVIEW

File: scripts/benchmark_k.py [hunk 1]
```diff
@@ -4,9 +4,11 @@
 import anndata
 import pandas as pd
 import gc
+import scanpy as sc
 
-sys.path.insert(1, './src/')
-from utils import *
+from markermap.utils import SmashPyWrapper, LassoNetWrapper, RandomBaseline
+from markermap.utils import MarkerMap, ConcreteVAE_NMSL, VAE_Gumbel_GlobalGate, VAE_l1_diag
+from markermap.utils import benchmark, plot_benchmarks
```
Comment (WilsonGregory) [inline scripts/benchmark_k.py:11]: Later, I am planning on splitting utils into 3 files: utils.py, vae_models.py, which will include that whole family of models, and then something like other_models.py which will include the SmashPyWrapper, LassoNetWrapper, and RandomBaseline and any other models that we add like that.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = markermap
+version = 0.0.1
+author = Wilson Gregory, Beel Sarwar
+author_email = wilson.g.gregory@gmail.com
```
Comment (WilsonGregory) [inline setup.cfg:5]: Let me know if you want your name to be included in the author list, and also if you want your email there. There can only be one author_email I believe if we are going to put it on pip, but we can add emails in the author line like `Wilson Gregory <yadayada@foo.com>`. We could also make a separate email and just give everyone access to it.

**Comment by WilsonGregory** [comment]
Gonna go ahead and merge this one.

**Comment by beelze-b** [comment]
Sorry I got to list late. Would you mind using my full name Nabeel Sarwar? 

Thanks

**Pull request merged**